### ISSUE TITLE
Chapter 12 – Custom Models and Training with TensorFlow

### ISSUE THREAD

**Issue by libingallin** [ageron/handson-ml2#63]
Hi~

I seem to have found a mistake.

The sentence above the cell "In [100]:" in [your jupyter notebook](https://github.com/ageron/handson-ml2/blob/master/12_custom_models_and_training_with_tensorflow.ipynb) is **"If you do the math, you will find that metric = loss * mean of sample weights (plus some floating point precision error)."**.

However, I found `loss = metric * mean_of_sample_weights` rather than `metric = loss *  mean_of_sample_weights` after I did the math. And this is verified by the code in [your jupyter notebook](https://github.com/ageron/handson-ml2/blob/master/12_custom_models_and_training_with_tensorflow.ipynb) (cell "In [102]:" and cell "In [124]:").

+ cell "In [102]:":
```python
In [102]: history.history["loss"][0], history.history["huber_fn"][0] * sample_weight.mean()
Out[102]: (0.11463345723687114, 0.11505695444753776)
```
+ cell "In [124]:"
```python
In [124]: history.history["loss"][0], history.history["HuberMetric"][0] * sample_weight.mean()
Out[124]: (0.36160108902965715, 0.36160092789449944)
```

**Comment by ageron** [comment]
Good catch, thanks! :)
